articles/deploying-nodejs-web-app/index.md

```diff
@@ -0,0 +1,143 @@
+
+### Deploying Your NodeJS Web App
+First, you’ll need to copy the code for your web app to your Droplet. If you’re using source control such as Git **Link to Section article on Git when it's published** then it’s a simple as installing git using `apt-get install git -y`  and then using the git clone command  `git clone (link to your repository)` and then add the link to your repository at the end.
+
+Second, you’ll need to install Node. Type:
+
+```bash
+curl -sL https://deb.nodesource.com/setup_14.x | sudo -E bash -
+sudo apt-get install -y nodejs
+```
+
+Third, you’ll need to install the node modules (dependencies) for your web app. If you installed all your modules with `-save` at the end which saves them to the package.json file then just type `npm install` and press enter.
+
+If not, when you run `npm start`  an error will appear with module not found. Type `npm install (module name)`  and press enter and then try running `npm start` again. Repeat the process until the error disappears.
+
+If you need to install MongoDB (if you’ve created a MongoDB database), then follow these [instructions](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-ubuntu/#install-mongodb-community-edition).
+
+Finally, type `npm start`  to start your web app. Now that your web app is running, in a new browser tab, type the IP Address of your droplet (found in the email that DigitalOcean sent when you created the droplet) followed by a colon and the port your app runs on. For example, `167.172.54.51:8080`.
+
+If you’re using an Express web server (which if you followed my **Link to previous developing NodeJS web app when it gets published**) you did, you’ll find the port number located in the  `app.listen()`  line inside the server.js file. For example,  `app.listen(8080)`  which is a common port used.
+
+Congratulations, your first NodeJS web app should be displayed in your web browser which is running on your DigitalOcean droplet.
+
+### Connecting Your Domain Name
+
+You typed in an IP Address and port number to view your web app but wouldn't you prefer a custom domain name like yourapp.com?
+
+Assuming you’ve already bought a domain, the first step is to connect it to your DigitalOcean droplet. If you’ve not, domain registrars like [Namecheap](https://www.namecheap.com) sell domain names and often other services such as email and static/CMS hosting (though it’s best to go with a dedicated hosting and email provider.)
+
+Login to your domain registrar and go to the advanced DNS settings of your domain. For example, on Namecheap, it’s the Advanced DNS tab on the Manage Domain screen.
+
+You want to add a new record as follows: the type should be set to A, the host should be either @ or blank depending on your provider and the value should be the IP Address of your droplet. Repeat the process for the host www which will do the same for the www version of your domain.
```

Comment:
You should add screenshots of navigating to the DNS settings and configuring the A record. I know from personal experience that this is confusing for many people and even if they aren't using Namecheap having a reference image will help.